src/components/calcite-value-list/calcite-value-list.tsx

```diff
@@ -202,6 +203,38 @@ export class CalciteValueList {
 
   getItemData = getItemData.bind(this);
 
+  keyDownHandler = (event) => {
+    const handleElement = event.composedPath().find((item) => {
+      return item.dataset && item.dataset.jsHandle;
+    });
+    // Only trigger keyboard sorting when the internal drag handle is focused and activated
+    if (!handleElement || !("activated" in handleElement.dataset)) {
+      return;
+    }
+    const value = event.target.value;
+    const startingIndex = this.items.findIndex((item) => {
+      return item.value === value;
+    });
+    let newIndex;
+    switch (event.key) {
+      case "ArrowUp":
+        newIndex = startingIndex - 1;
+        newIndex = newIndex < 0 ? this.items.length - 1 : newIndex;
+        break;
+      case "ArrowDown":
+        newIndex = startingIndex + 1;
+        newIndex = newIndex > this.items.length - 1 ? 0 : newIndex;
+        break;
+      default:
+        return;
+    }
+    const order = this.sortables[0].toArray();
```

Comment:
question: can there be multiple sortables?

Reply:
Originally, when the architecture needed to support subgroups, I had to create a sortable for each group, but group them using the group prop. Since that requirement was dropped, The heirarchy flattened and there's only one now.